cypress/README.md

```diff
@@ -54,10 +54,11 @@ TODO allow using {id:...} instead of string label for menu items, gtl items, tre
 
 #### GTL
 
-* `cy.gtl()` - return GTL
 * `cy.gtl_error` - check that error message is present
 * `cy.gtl_no_record` - check that `No data` message is present
-* `cy.gtl_click('Name')` - click on a GTL
+* `cy.gtlGetTable` - return GTL table
+* `cy.gtlGetRows(columns)` - return GTL table row data in an array. columns: Array of ints of columns to read. Example: [1,2,3] will return all row data from columns 1,2 and 3.
```

Comment:
Is this index value 0-based or 1-based?

Reply:
Assuming it's 0-based, I think this might be a nice way to present that:

```suggestion
* `cy.gtlGetRows(columns)` - return GTL table row data in an array. columns: Array of 0-based indexes of the columns to read (e.g. [1, 2, 3] will return all row data from columns 1, 2, and 3).
```

Reply:
I also noticed getRows can be called without the parameter, so we may want to add something like "Default value for columns is to return all columns."

Reply:
Default value right now is column 1 (usually the name/description column). Should I change that?

Reply:
> Is this index value 0-based or 1-based?

@Fryguy It is 0 based but 0 is the checkbox column for this table.

Reply:
ah ok - makes sense.  We may want to say 1-based, then?

Reply:
Ok, then I think we should still say 0-based in the comment.

Reply:
> Actually would it make more sense to just remove the default parameter and just always have the user input the columns they want to read? This might be better since we wouldn't have to find the number of columns for the table

Yeah that's better, because the caller (the test) is really the only one that knows the "correct" column

Reply:
Sounds good, made the fix and re-pushed